src/annotation.js

```diff
@@ -1152,8 +1154,18 @@ function continuousVerticesProcessAction(m_this, evt, name) {
       }
       vertices[vertices.length - 1] = evt.mouse.mapgcs;
       vertices.push(evt.mouse.mapgcs);
-      return true;
+      update = true;
     }
+    if ((ccp || ccp === 0) && evt.event === geo_event.actionup &&
+        (cpp === true || layer.displayDistance(vertices[0], null, evt.mouse.map, 'display') <= cpp)) {
```

Comment:
Should this be `ccp === true` ?

Right now, it works but I have to set continuousPointProximity to true as well.

Reply:
Yes, my mistake.